Problem: hax fails to start on the 2nd node

When bootstrap-ping on multi-node (in particular, two nodes) `hax` fails like this:

```
Sep 20 04:41:36 sage75c1 systemd[1]: Starting HAX daemon for Hare...
Sep 20 04:41:37 sage75c1 sh[77343]: 2019-09-20 04:41:37,163 [DEBUG] {MainThread} Starting new HTTP connection (1): 127.0.0.1:8500
Sep 20 04:41:37 sage75c1 sh[77343]: 2019-09-20 04:41:37,167 [DEBUG] {MainThread} http://127.0.0.1:8500 "GET /v1/agent/self HTTP/1.1" 200 None
Sep 20 04:41:37 sage75c1 sh[77343]: 2019-09-20 04:41:37,174 [DEBUG] {MainThread} http://127.0.0.1:8500 "GET /v1/catalog/service/hax HTTP/1.1" 200 383
Sep 20 04:41:37 sage75c1 sh[77343]: 2019-09-20 04:41:37,180 [DEBUG] {MainThread} http://127.0.0.1:8500 "GET /v1/catalog/service/hax HTTP/1.1" 200 383
Sep 20 04:41:37 sage75c1 sh[77343]: 2019-09-20 04:41:37,191 [DEBUG] {MainThread} http://127.0.0.1:8500 "GET /v1/agent/self HTTP/1.1" 200 None
Sep 20 04:41:37 sage75c1 sh[77343]: 2019-09-20 04:41:37,197 [DEBUG] {MainThread} http://127.0.0.1:8500 "GET /v1/catalog/service/hax HTTP/1.1" 200 383
Sep 20 04:41:37 sage75c1 sh[77343]: 2019-09-20 04:41:37,209 [DEBUG] {MainThread} http://127.0.0.1:8500 "GET /v1/agent/self HTTP/1.1" 200 None
Sep 20 04:41:37 sage75c1 sh[77343]: 2019-09-20 04:41:37,220 [DEBUG] {MainThread} http://127.0.0.1:8500 "GET /v1/catalog/service/hax HTTP/1.1" 200 383
Sep 20 04:41:37 sage75c1 sh[77343]: 2019-09-20 04:41:37,229 [DEBUG] {MainThread} http://127.0.0.1:8500 "GET /v1/agent/self HTTP/1.1" 200 None
Sep 20 04:41:37 sage75c1 sh[77343]: 2019-09-20 04:41:37,244 [DEBUG] {MainThread} http://127.0.0.1:8500 "GET /v1/catalog/service/hax HTTP/1.1" 200 383
Sep 20 04:41:37 sage75c1 sh[77343]: 2019-09-20 04:41:37,246 [DEBUG] {MainThread} Loading library from path: /home/ant/hare/.env/lib64/python3.6/site-packages/hax/../libhax.cpython-36m-x86_64-linux-gnu.so
Sep 20 04:41:37 sage75c1 sh[77343]: 2019-09-20 04:41:37,359 [INFO] {qconsumer} Handler thread has started
Sep 20 04:41:37 sage75c1 sh[77343]: 2019-09-20 04:41:37,359 [DEBUG] {MainThread} Starting m0_halon_interface
Sep 20 04:41:37 sage75c1 sh[77343]: 2019-09-20 04:41:37,360 [DEBUG] {qconsumer} Waiting for the next message
Sep 20 04:41:37 sage75c1 sh[77343]: 2019-09-20 04:41:37,521 [DEBUG] {Dummy-1} Received entrypoint request from remote endpoint '192.168.180.166@tcp:12345:1:1', process fid = 0x7200000000000001:0x33 The request will be processed in another thread.
Sep 20 04:41:37 sage75c1 sh[77343]: 2019-09-20 04:41:37,562 [DEBUG] {qconsumer} Got something from the queue
Sep 20 04:41:37 sage75c1 sh[77343]: 2019-09-20 04:41:37,562 [DEBUG] {qconsumer} Processing entrypoint request from remote endpoint '192.168.180.166@tcp:12345:1:1', process fid 0x7200000000000001:0x33
Sep 20 04:41:37 sage75c1 sh[77343]: 2019-09-20 04:41:37,567 [DEBUG] {qconsumer} Starting new HTTP connection (1): 127.0.0.1:8500
Sep 20 04:41:37 sage75c1 sh[77343]: 2019-09-20 04:41:37,573 [DEBUG] {qconsumer} http://127.0.0.1:8500 "GET /v1/kv/leader HTTP/1.1" 200 148
Sep 20 04:41:37 sage75c1 sh[77343]: 2019-09-20 04:41:37,579 [DEBUG] {qconsumer} http://127.0.0.1:8500 "GET /v1/session/info/b23ca946-db08-7983-141b-f93fcbbb2118 HTTP/1.1" 200 177
Sep 20 04:41:37 sage75c1 sh[77343]: 2019-09-20 04:41:37,593 [DEBUG] {qconsumer} http://127.0.0.1:8500 "GET /v1/catalog/service/confd HTTP/1.1" 200 384
Sep 20 04:41:37 sage75c1 sh[77343]: 2019-09-20 04:41:37,596 [DEBUG] {qconsumer} Passing the entrypoint reply to hax.c layer
Sep 20 04:41:37 sage75c1 sh[77343]: 2019-09-20 04:41:37,597 [DEBUG] {qconsumer} Entrypoint request has been replied to
Sep 20 04:41:37 sage75c1 sh[77343]: 2019-09-20 04:41:37,597 [DEBUG] {qconsumer} Waiting for the next message
Sep 20 04:41:37 sage75c1 sh[77343]: 2019-09-20 04:41:37,612 [ERROR] {MainThread} Cannot start ha_link. m0_halon_interface::start returned 0
Sep 20 04:41:37 sage75c1 sh[77343]: 2019-09-20 04:41:37,612 [DEBUG] {MainThread} Destroying ha_link
Sep 20 04:41:37 sage75c1 sh[77343]: mero[77343]:  fb80  FATAL  [addb2/addb2.c:1094:m0_addb2__mach_print]  mach: 0x7f32100008c0 "m0_loc_thread" "".
Sep 20 04:41:37 sage75c1 sh[77343]: mero[77343]:  fa80  FATAL  [lib/assert.c:48:m0_panic]  panic: sys->sy_total == 0 at m0_addb2_sys_fini() (addb2/sys.c:255)  [git: v1.4-323-g0fd3693] //m0trace.77343
Sep 20 04:41:37 sage75c1 sh[77343]: Mero panic: sys->sy_total == 0 at m0_addb2_sys_fini() addb2/sys.c:255 (errno: 0) (last failed: none) [git: v1.4-323-g0fd3693] pid: 77343  //m0trace.77343
Sep 20 04:41:37 sage75c1 sh[77343]: Mero panic reason: 1
Sep 20 04:41:37 sage75c1 sh[77343]: /home/ant/hare/hax/../../mero/mero/.libs/libmero.so.1(m0_arch_backtrace+0x20)[0x7f32369f0810]
Sep 20 04:41:37 sage75c1 sh[77343]: /home/ant/hare/hax/../../mero/mero/.libs/libmero.so.1(m0_arch_panic+0xe6)[0x7f32369f09c6]
Sep 20 04:41:37 sage75c1 sh[77343]: /home/ant/hare/hax/../../mero/mero/.libs/libmero.so.1(+0x35d654)[0x7f32369e0654]
Sep 20 04:41:37 sage75c1 sh[77343]: /home/ant/hare/hax/../../mero/mero/.libs/libmero.so.1(+0x2792ca)[0x7f32368fc2ca]
Sep 20 04:41:37 sage75c1 sh[77343]: /home/ant/hare/hax/../../mero/mero/.libs/libmero.so.1(m0_subsystems_fini+0x2a)[0x7f32369fb05a]
Sep 20 04:41:37 sage75c1 sh[77343]: /home/ant/hare/hax/../../mero/mero/.libs/libmero.so.1(m0_module_fini+0x76)[0x7f3236a058d6]
Sep 20 04:41:37 sage75c1 sh[77343]: /home/ant/hare/hax/../../mero/mero/.libs/libmero.so.1(m0_halon_interface_fini+0x67)[0x7f32369c9f47]
Sep 20 04:41:37 sage75c1 sh[77343]: /home/ant/hare/.env/lib64/python3.6/site-packages/hax/../libhax.cpython-36m-x86_64-linux-gnu.so(destroy_halink+0x29)[0x7f323780d279]
Sep 20 04:41:37 sage75c1 sh[77343]: /lib64/libffi.so.6(ffi_call_unix64+0x4c)[0x7f323c755dcc]
Sep 20 04:41:37 sage75c1 sh[77343]: /lib64/libffi.so.6(ffi_call+0x1f5)[0x7f323c7556f5]
Sep 20 04:41:37 sage75c1 sh[77343]: /usr/lib64/python3.6/lib-dynload/_ctypes.cpython-36m-x86_64-linux-gnu.so(_ctypes_callproc+0x502)[0x7f323c974862]
Sep 20 04:41:37 sage75c1 sh[77343]: /usr/lib64/python3.6/lib-dynload/_ctypes.cpython-36m-x86_64-linux-gnu.so(+0x9746)[0x7f323c96d746]
Sep 20 04:41:37 sage75c1 sh[77343]: /lib64/libpython3.6m.so.1.0(_PyObject_FastCallDict+0x90)[0x7f324435f640]
Sep 20 04:41:37 sage75c1 sh[77343]: /lib64/libpython3.6m.so.1.0(+0x151bec)[0x7f3244408bec]
Sep 20 04:41:37 sage75c1 sh[77343]: /lib64/libpython3.6m.so.1.0(_PyEval_EvalFrameDefault+0x3a7)[0x7f32443fd0a7]
Sep 20 04:41:37 sage75c1 sh[77343]: /lib64/libpython3.6m.so.1.0(+0x15182a)[0x7f324440882a]
Sep 20 04:41:37 sage75c1 sh[77343]: /lib64/libpython3.6m.so.1.0(+0x151b63)[0x7f3244408b63]
Sep 20 04:41:37 sage75c1 sh[77343]: /lib64/libpython3.6m.so.1.0(_PyEval_EvalFrameDefault+0x3a7)[0x7f32443fd0a7]
Sep 20 04:41:37 sage75c1 sh[77343]: /lib64/libpython3.6m.so.1.0(+0x15182a)[0x7f324440882a]
Sep 20 04:41:37 sage75c1 sh[77343]: /lib64/libpython3.6m.so.1.0(+0x151b63)[0x7f3244408b63]
Sep 20 04:41:37 sage75c1 sh[77343]: /lib64/libpython3.6m.so.1.0(_PyEval_EvalFrameDefault+0x3a7)[0x7f32443fd0a7]
Sep 20 04:41:37 sage75c1 sh[77343]: /lib64/libpython3.6m.so.1.0(PyEval_EvalCodeEx+0x1b2)[0x7f3244408e62]
Sep 20 04:41:37 sage75c1 sh[77343]: /lib64/libpython3.6m.so.1.0(PyEval_EvalCode+0x3b)[0x7f3244409afb]
Sep 20 04:41:37 sage75c1 sh[77343]: /lib64/libpython3.6m.so.1.0(+0x1dc1ee)[0x7f32444931ee]
Sep 20 04:41:37 sage75c1 sh[77343]: /lib64/libpython3.6m.so.1.0(PyRun_FileExFlags+0xa4)[0x7f324433e464]
Sep 20 04:41:37 sage75c1 sh[77343]: /lib64/libpython3.6m.so.1.0(PyRun_SimpleFileExFlags+0x39c)[0x7f324433e836]
Sep 20 04:41:37 sage75c1 systemd[1]: hax.service: main process exited, code=killed, status=6/ABRT
Sep 20 04:41:37 sage75c1 sh[77343]: /lib64/libpython3.6m.so.1.0(Py_Main+0x652)[0x7f32444996a2]
Sep 20 04:41:37 sage75c1 sh[77343]: /home/ant/hare/.env/bin/python3(main+0x119)[0x400a99]
Sep 20 04:41:37 sage75c1 sh[77343]: /lib64/libc.so.6(__libc_start_main+0xf5)[0x7f32435e7445]
Sep 20 04:41:37 sage75c1 sh[77343]: /home/ant/hare/.env/bin/python3[0x400c20]
Sep 20 04:41:37 sage75c1 systemd[1]: Unit hax.service entered failed state.
```

To reproduce: `./bootstrap cfgen/_misc/two-nodes.yaml`. Commit 7922d19.

--- a/hax/hax/halink.py
+++ b/hax/hax/halink.py
@@ -40,9 +40,10 @@ class HaLink:
         result = self._ffi.start(self._ha_ctx, make_c_str(rpc_endpoint),
                                  process.to_c(), ha_service.to_c(),
                                  rm_service.to_c())
-        if not result:
-            logging.error('Cannot start ha_link. m0_halon_interface::start' +
-                          ' returned 0')
+        if result:
+            logging.error(
+                'Cannot start ha_link. m0_halon_interface::start' +
+                ' returned non-zero code (%s)', result)
             raise RuntimeError('Cannot start m0_halon_interface.' +
                                'Please check mero logs for more details.')
 

